app/templates/_layout.html

```diff
@@ -103,3 +103,7 @@
     }
   }) }}
 {% endblock %}
+
+{% block bodyEnd %}
+  <script async type="text/javascript" src="{{ '/alerts/assets/javascripts/govuk-frontend-skip-link.js' | file_fingerprint }}"></script>
```

Comment:
There's actually no reason to serve the skiplink JS async if it's at the bottom of the page as it can't block anything in that position. I think moving it up, as per your other comment, will make it clearer the value of using async on that script but not these ones.